client.go

```diff
@@ -716,6 +716,11 @@ func (c *Client) IsProxySet() bool {
 	return c.proxyURL != nil
 }
 
+// GetClient method returns the current http.Client used by the resty client. 
+func (c *Client) GetClient() *http.Client {
```

Comment:
The same `GetClient` method required for default client too at this file [default.go](https://github.com/go-resty/resty/blob/master/default.go). Could you please add it?

```go
// GetClient method returns the current `http.Client` used by the default resty client.
func GetClient() *http.Client {
  return DefaultClient.httpClient
}
```

And adding test coverage for these two methods.

Reply:
+1, can do

Reply:
Thank you @jfarrell